scripts/version-generator/main.go

```diff
@@ -0,0 +1,56 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"path/filepath"
+
+	"github.com/ActiveState/cli/internal/condition"
+	"github.com/pkg/errors"
+)
+
+func printUsage() {
+	fmt.Println("")
+	fmt.Println("<branch> <outputPath> The path to output version.json to")
+}
+
+func main() {
+	if !condition.InTest() {
+		run()
+	}
+}
+
+func run() {
+	flag.Parse()
+	if flag.NArg() != 2 {
+		printUsage()
+		os.Exit(0)
+	}
+	branch := flag.Arg(0)
+
+	stateURL := "https://s3.ca-central-1.amazonaws.com/cli-update/update/state/%s/linux-amd64.json"
```

Comment:
Please drop "linux", this is no longer platform specific. You can just go onto S3 and copy the linux file to "version.json".